cmake/MiNiFiOptions.cmake

```diff
@@ -102,9 +101,8 @@ add_minifi_option(ENABLE_SQL "Enables the SQL Suite of Tools." ON)
 add_minifi_option(ENABLE_MQTT "Enables the mqtt extension." OFF)
 add_minifi_option(ENABLE_PCAP "Enables the PCAP extension." OFF)
 add_minifi_option(ENABLE_LIBRDKAFKA "Enables the librdkafka extension." ON)
-add_minifi_option(ENABLE_SCRIPTING "Enables the scripting extensions." OFF)
 add_minifi_option(ENABLE_LUA_SCRIPTING "Enables lua scripting" OFF)
-add_minifi_option(DISABLE_PYTHON_SCRIPTING "Disables python scripting" OFF)
+add_minifi_option(ENABLE_PYTHON_SCRIPTING "Enables python scripting" OFF)
```

Comment:
was changing the default intentional?

Reply:
I see that there was an `ENABLE_PYTHON`, `ENABLE_SCRIPTING` and `DISABLE_PYTHON_SCRIPTING` option, did their defaults have the same effect?

Reply:
There was ENABLE_PYTHON which controlled the python bindings in nanofi. We didnt really use that one so I've removed the whole nanofi-python bindings entirely.

Prior to this PR, there was ENABLE_SCRIPTING which was OFF by default, and there was two additional flags namely DISABLE_PYTHON_SCRIPTING(OFF), ENABLE_LUA_SCRIPTING(OFF), which only mattered if the ENABLE_SCRIPTING was ON.

Instead of this I've added ENABLE_PYTHON_SCRIPTING ~(ENABLE_SCRIPTING && NOT DISABLE_PYTHON_SCRIPTING) and ENABLE_LUA_SCRIPTING ~(ENABLE_SCRIPTING && ENABLE_LUA_SCRIPTING), so long story short by default both lua and python was disabled and they still are disabled by default.